.coderabbit.yaml

```
@@ -1,16 +1,16 @@
-  high_level_summary: true
-  review_status: true
-  collapse_walkthrough: false
-  auto_review:
-    enabled: true
-    drafts: false
-  finishing_touches:
-    unit_tests:
-      enabled: false
-chat:
-  auto_reply: true
+language: "ko-KR"
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)